### ISSUE TITLE
Stale metrics doesn't dissapear   

### ISSUE THREAD

**Issue by oOHenry** [VictoriaMetrics/VictoriaMetrics#383]
**Describe the bug**
I have a Metric which is exported by 5 targets. It exists only for short time. In the example: `ceph_exporter_pg_states_combined{state="peering"}` exists only a short time. When the ceph cluster is peering.
When I query the last 3 hours I get the expected result:
![image](https://user-images.githubusercontent.com/16702685/77412362-131b6e00-6dbe-11ea-9da9-02925121804e.png)
But when I query the last 12 hours the metrics appear much longer:
![image](https://user-images.githubusercontent.com/16702685/77412447-2e867900-6dbe-11ea-8955-1094066cb4a1.png)
Maybe related to: #382 

**Version**
```
$ vmselect --version
vmselect-20200303-200611-tags-v1.34.2-cluster-0-g50190263
```

**Comment by valyala** [comment]
VictoriaMetrics fills gaps between actual data points on the graph if they are smaller than 2x of median interval between data points, since it assumes the time series exists for up to 2x scrape intervals after the actual data point. It looks like you hit the corner case, where `ceph_exporter_pg_states_combined{state="peering"}` time series contains data points with big median interval between them. This issue can be solved in [v1.34.4](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) by setting `-search.maxStalenessInterval` command-line flag to value, which exceeds the maximum scrape interval for all the stored time series.

For instance, if the maximum scrape interval for all the time series stored in VictoriaMetrics is 5 minutes, then `-search.maxStalenessInterval=10m` should fix your case.

**Comment by f41gh7** [comment]
Seems like, issue was solved. Feel free to re-open or create new issue.
